src/components/widgets/CheckboxWidget.js

```diff
@@ -13,13 +13,18 @@ function CheckboxWidget({
 }) {
   return (
     <div className={`checkbox ${disabled ? "disabled" : ""}`}>
+      { schema.description
+        ? <DescriptionField description={ schema.description }/>
+        : null
+      }
       <label>
         <input type="checkbox"
           id={id}
           checked={typeof value === "undefined" ? false : value}
           required={required}
           disabled={disabled}
           autoFocus={autofocus}
+          title={schema.description}
```

Comment:
The title attribute should be set on the `<label>` element, so the tooltip is shown when hovering over the whole widget.